### DISCUSSION TITLE
New function to display tracks of my directory in the area displayed on the screen

### DISCUSSION THREAD

**Discussion by bobeeeze** [osmandapp/OsmAnd#13683] [Ideas]
Hello everybody!
I have been using Osmand for several years now, mainly for hiking, and there is a function that I miss. It would be a button to display the tracks of my directory that are entirely or partially in the area displayed on the screen. I often want to know where I've been the other times I've been to a place. Sometimes I don't even remember if I've been there before.
This button could open a list of available tracks at this location and allow me to choose and select the ones I want to display, either all or some.
So I think it would be really nice to have this function in Osmand.
I hope you understood what I meant, because my English comes from a translator.
Thank you Osmand team for your great application

**Comment by tommycrock** [comment]
Sounds like a good idea. Perhaps a filter button in the top right corner of the "My places" dialog with an "In view" option. I'm now wondering what other useful filters there might be. Track or route? Kind of favourite POI? Icon /colour?

**Comment by vshcherb** [comment]
Yes we plan to develop tracks heatmap feature, so you could enable all tracks and browse them.

**Reply by bobeeeze** [reply]
Are you still planning to do this function?

**Reply by vshcherb** [reply]
Yes, now it's possible to create obf file with all tracks displayed same time, though  we also plan next year to have proper heatmap created

**Reply by kuiperonline** [reply]
Anything new allready?
